@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -481,6 +481,10 @@ export interface TerrainTechnique extends MakeTechniqueAttrs<TerrainTechniquePar
     name: "terrain";
 }
 
+export interface LabelRejectionLineTechnique extends MakeTechniqueAttrs<BaseTechniqueParams> {
+    name: "label-rejection-line";
```

Comment:
Sorry for bringing this up so late (and not in the solutioning) but why do we need a separate technique?
This could also be just a property in the `SolidLineTechnique`. We could even reuse the buffer that is created by the `SolidLineTechnique`

Reply:
Good point, never occurred to me, but makes complete sense, will look into it.

Reply:
I brought back the technique because it is needed to have a separate datasource with lower LOD.